### ISSUE TITLE
stops functioning 

### ISSUE THREAD

**Issue by quasi8** [compvid30/scriptblock#11]
Running scriptblock 1.3
Windows 7 Pro Chrome 51.0.2693.2 64-bit dev 
Windows 8.1 Pro Chrome 51.0.2693.2 64-bit dev

Tried clearing all caches still same thing and installed in new chrome profile same thing

![sb](https://cloud.githubusercontent.com/assets/17346852/14175625/3120079a-f71a-11e5-99df-04dc4bf113f7.PNG)

**Comment by compvid30** [comment]
Hi,

just checked that in Version 51.0.2695.1 canary (64-bit). Everything works like normal.

Happens from time to time that something does not work propperly in the dev versions. Best use the stable versiones. ;)

**Comment by quasi8** [comment]
Yes I am a long time user from notscripts days but always have been running dev and has been very stable since then and was thinking this might be an indicator of an upcoming issue in the stable channel.

**Comment by compvid30** [comment]
Version 50.0.2661.75 m is out and it works. Hope they fix it for 51 too.

**Comment by sobitthen** [comment]
Are there dev versions here, new to this page, trying to resolve the greyed out icon issue where websites can bypass this extension.. OIC it was dev on chrome itself.. unsure what to do here, how t use the files, field engineer not a coder...
